servers/rendering/renderer_rd/storage_rd/render_scene_buffers_rd.cpp

```diff
@@ -387,6 +398,10 @@ const RD::TextureFormat RenderSceneBuffersRD::get_texture_format(const StringNam
 }
 
 RID RenderSceneBuffersRD::get_texture_slice(const StringName &p_context, const StringName &p_texture_name, const uint32_t p_layer, const uint32_t p_mipmap, const uint32_t p_layers, const uint32_t p_mipmaps) {
+	return get_texture_slice_view(p_context, p_texture_name, p_layer, p_mipmap, p_layers, p_mipmaps, RD::TextureView());
+}
+
+RID RenderSceneBuffersRD::get_texture_slice_view(const StringName &p_context, const StringName &p_texture_name, const uint32_t p_layer, const uint32_t p_mipmap, const uint32_t p_layers, const uint32_t p_mipmaps, RD::TextureView p_view) {
```

Comment:
You also need to add a `_get_texture_slice_view` which uses `Ref<RDTextureView>` and add a bind method so this is accessible externally.

Have a look at `_create_texture_view` that does the same for `create_texture_view`.

Reply:
Sounds fine to me, wasn't aware if we wanted a new API addition as I worked my way around modifying the existing one to keep compatibility.

Reply:
Alright, we'll add the extra function.

Reply:
Adding is fine, modifying is a problem :)